### PR TITLE
Setting up hello world repo

### PR DESCRIPTION
**Pull request by TheLuisAbarca**
### **Setting up hello world repo**

In this exercise I made the following:
 
1.     Initialized a repo.
2.     Created a new branch.
3.     Added a .gitignore file.
4.     Set up a linter for HTML and CSS and made sure that it was ignored by git.
4.     Added a descriptive README file to my project
6.     Added 3 files (each one in a separate commit):
7.         A .html file that includes a header with the text "Hello Microverse!" using h1 tag .
8.         A .css file that includes one class with styles for my header in my favorite color.
9.         A test.md file with the text "This file should be ignored by git".
10.     Made sure that test.md was actually ignored by git and was not present in your GitHub repository.
11.     Opened a pull request.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+test.md
+*.yml
+.hintrc
+.stylelintrc.json
```
Comment (Ghiftee) [inline .gitignore]: - [ ] Please remove these files from the .gitignore, they are not meant to be ignored. You need them for your linters to pass.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,15 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Hello World
 
-# Project Name
-
-> One paragraph statement about the project.
+> Repo where I practice the rules and correct behavior when 'Creating Repos' comes to action.
```
Comment (Ghiftee) [inline README.md]: - [ ] Kindly edit this line to provide a brief summary of what the project is about.

**Review by Ghiftee — CHANGES_REQUESTED** [review]
Hello 👋🏽  @TheLuisAbarca 

Good job working on the project so far! 
✔️  You have a descriptive PR summary and title.

However, there are some issues that you need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

- [ ] Your linters are currently not passing, one of the reasons is because the .hintrc and .stylelintrc.json files are empty, please add their contents and check that the linters pass. You can find their content [here](https://github.com/microverseinc/linters-config/tree/master/html-css).
- [ ] Also please remove the .hintrc and .stylelintrc.json files from the .gitignore file, they are not meant to be ignored. The only files that should be in the gitignore file are test.md and node_modules.
- [ ] The node_modules file should be added to .gitignore as it can be added by accident while installing linters and it could mess up your git operations and Github repo.
- [ ] Kindly update the screenshot on your README with a screenshot of how your project looks when viewed on a browser. This would give a more professional feel to your README.
- [ ] Good job with your commit messages. However, going forward I would suggest being more consistent. A few of your commit messages are not imperative and capitalized. For example:

- 'Added .gitignore' should be 'Add .gitignore'.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .gitignore (hunk 1 above)
Comment (TheLuisAbarca) [reply .gitignore]: Ok, got it @Ghiftee, It is done on commit a1bb727bce321ac730dd1469a835c47df9e557d4

File: README.md (hunk 2 above)
Comment (TheLuisAbarca) [reply README.md]: Ok, got it @Ghiftee, It is done on commit 2c4e8e2304c54fe7a22b8dc6257f5ae3b7521a0e

**Review by mikethreels — APPROVED** [review]
Hi Luís, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**